test/corpus/todo.txt

```diff
@@ -388,3 +282,28 @@ proc.call('l') # => 2
     (argument_list
       (char)))
   (comment))
+
+=================
+fun pointer param
+=================
+
+lib LibC
+  fun strerror(Int, Char*, SizeT) : Int
```

Comment:
Can't seem to figure out why `fun_type_param_list` won't accept the pointer type

Reply:
Was able to figure it out